### DISCUSSION TITLE
Looking for example for atsamc21 CAN FD

### DISCUSSION THREAD

**Discussion by silverv** [zephyrproject-rtos/zephyr#56481] [General]
@benys Do you have an example to share? I didn't find one in the pull request: https://github.com/zephyrproject-rtos/zephyr/pull/50408

**Comment by benys** [comment]
@silverv my PR has only CAN driver for C21 and extension for SAMC21N board.
I don't have expirence with CANFD - I used only CAN.
To confirm my solution I check samples from samples/drivers/can and with in my project.
